frontend/src/app/supportAdmin/ManageUsers/AdminManageUser.test.tsx

```diff
@@ -55,18 +70,29 @@ const validResponse = [
     },
   },
 ];
-const searchForValidUser = async () => {
-  const searchInput = screen.getByLabelText("Search by email address of user");
-  fireEvent.change(searchInput, { target: { value: "ben@example.com" } });
-  fireEvent.click(screen.getByAltText("Search"));
+const searchForValidUser = async (
+  user: UserEvent,
+  email = "ben@example.com",
+  fullName = "Barnes, Ben Billy"
+) => {
+  const searchInput = await screen.findByLabelText(
+    "Search by email address of user"
+  );
+  await act(async () => {
```

Comment:
johanna and i chatted about prioritizing [the `act` spike ticket](https://github.com/orgs/CDCgov/projects/15/views/1?filterQuery=spike&pane=issue&itemId=37752857) tomorrow in planning and going back and fixing these as part of that effort